### PR TITLE
Update translations for pt_BR

### PR DESCRIPTION
**Pull request by prschopf**
Im new in Github as contributor and Im not sure if I did it right.

I have changed this file: airsonic-main/src/main/resources/org/airsonic/player/i18n/ResourceBundle_pt_BR.properties and created a pull request to have it published.

Is it right? I really dont know what to do to have it published. Please, help me with that.

### CODE REVIEW

**Comment by fxthomas** [comment]
You actually did great, this is exactly how it's supposed to be done. Congratulations on your first pull request :smile: 

I took the liberty to update the title to reflect what you want to merge — in general try to have a descriptive title so that we can more easily see what your PR does in the list.

File: airsonic-main/src/main/resources/org/airsonic/player/i18n/ResourceBundle_pt_BR.properties [hunk 1]
```diff
@@ -1,736 +1,735 @@
...
-helppopup.ignoredarticles.title=Articles to ignore
-helppopup.ignoredarticles.text=Allows you to specify a list of articles (such as "The") that will be ignored when creating the index.
-helppopup.shortcuts.title=Shortcuts
-helppopup.shortcuts.text=A space-separated list of top-level folders to create shortcuts to. Use quotes to group words, for instance:</p> <p><em>New Incoming "Sound tracks"</em>
-helppopup.language.title=Language
-helppopup.language.text=Language used in Airsonic.
-helppopup.visibility.title=Visibility
-helppopup.visibility.text=Select which details should be displayed for each song.
-helppopup.partymode.title=Party mode
-helppopup.partymode.text=When party mode is enabled, the user interface is simplified and easier to operate
```
Comment (fxthomas) [inline airsonic-main/src/main/resources/org/airsonic/player/i18n/ResourceBundle_pt_BR.properties]: Change to `sAMAccountName='{0}'` like the original.

**Comment by prschopf** [comment]
Sorry for my noobish. I did some updates in this file. Should I reopen this pull request? Is it ok in this way?

**Comment by fxthomas** [comment]
Thanks, that worked out great! I merged it on master, your changes will be live on the next release :wink:

**Pull request merged**